arm-datalake-analytics/job/2016-11-01/examples/AdlPipelineGet.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parameters": {
+    "api-version": "2016-11-01",
+    "pipelineId": "076713da-9018-41ae-a3bd-9eab14e54d09",
```

Comment:
this parameter is called pipelineIdentity in the spec file https://github.com/Azure/azure-rest-api-specs/pull/1272/files#diff-c3357f2ab384c3b8ab52d353d8b2724eR94 (https://travis-ci.org/Azure/azure-rest-api-specs/jobs/238436043#L1113)